remote-workstations/new-agent-vm/Install-PCoIPAgent.sh

```diff
@@ -243,131 +238,326 @@ active_d = ldap_lib(args.address, args.user_name, args.password, args.domain)
 active_d.add_computer_to_group(args.computer, args.group)
 
 EOF
-sudo python $file_path -d "$DOMAIN_NAME" -a "$DC_ADDRESS" -u "$USERNAME" -p "$PASSWORD" -c "$VM_NAME" -g "$GROUP"
 
+    sudo python $file_path -d "$DOMAIN_NAME" -a "$DC_ADDRESS" -u "$USERNAME" -p "$PASSWORD" -c "$VM_NAME" -g "$GROUP"
+}
+
+install_gui() 
+{
+    # Make sure Linux OS is up to date
+    echo "--> Updating Linux OS to latest"
+
+    sudo yum -y update  # --exclude=WALinuxAgent
+
+    # Install Desktop
+    echo "-->Install desktop"	
+    #sudo yum -y groupinstall 'X Window System' 'GNOME'
```

Comment:
There are several commented-out commands or debug statements in this function that I think we could just remove for the sake of clarity. :)

Reply:
let's keep those comments here